packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -139,7 +139,8 @@ export class ConnectionPropsForSessCfg {
             promptForValues.push("hostname");
         }
 
-        if (ConnectionPropsForSessCfg.propHasValue(sessCfgToUse.port) === false && !doNotPromptForValues.includes("port")) {
+        if ((ConnectionPropsForSessCfg.propHasValue(sessCfgToUse.port) === false || sessCfgToUse.port === 0) &&
+            !doNotPromptForValues.includes("port")) {
```

Comment:
clever way to prompt for port 🥳 

Question:
How would Zowe Explorer react to this new prompt item?

Curious if you could share a screenshot or something like that 😋 

Happy to approve afterwards this ⬆️

Reply:
Zowe Explorer doesn't use AddPropsOrPrompt as far as Trae and I are aware. As for ZE, its current behavior is such that if a port is 0 in the team config, it tries using port 443 (for z/OSMF at least).

Reply:
For the current CICS plug-in, a value of 0 results in an error.
![image](https://github.com/zowe/zowe-cli/assets/53578529/e08faa71-a112-492a-aad2-729c8d647b51)

Reply:
thanks for providing more details around ZE (and related apps) 😋